basics/4_document_generators/app.py

```diff
@@ -28,6 +28,6 @@
 with flow:
     flow.index(inputs=docs)
     query = Document(text=input("Please enter your search term: "))
-    response = flow.search(inputs=query, return_results=True)
+    response = flow.search(inputs=query)
```

Comment:
Might need to add `{'traversal_paths': 'r'}` in `uses_with` to get this to run in the meantime until spacy encoder is migrated to Jina 3. Same for example 5, 6 and 7

Reply:
Ok!